packages/app/src/systems/Overlay/components/OverlayDialog/OverlayDialog.tsx

```diff
@@ -26,6 +37,7 @@ const styles = {
     height: WALLET_HEIGHT,
     maxWidth: WALLET_WIDTH,
     maxHeight: 'none',
+    background: '#0C0E0D',
```

Comment:
> Inside the popup, the content has a different color than the background of the regular popups, making the cards mix with the background.
@luizstacio - To fix the above issue, made the Overlay background darker than `$gray1`, let me know if there's a better color to use here instead.